sites/firehouse.com/server/graphql/fragments/content-list-expanded.js

```diff
@@ -0,0 +1,66 @@
+const gql = require('graphql-tag');
+
+module.exports = gql`
+
+fragment WebsiteContentListExpandedFragment on Content {
```

Comment:
Verify which fields are actually used.  Rename this file to be declarative of what's using it, instead of a quasi-generic name.